### DISCUSSION TITLE
Ubuntu 22.04 on WSL

### DISCUSSION THREAD

**Discussion by Nivethanb** [microsoft/WSL#11180] [Q&A]
How to get Ubuntu 22.04 tar image to import on my WSL.

**Comment by lane-ftw** [comment]
`wsl –import <Image Name you choose> <Directory to store and run the image> <Directory location of the exported .tar file>`
`wsl -d <WSL imported image name>`
So if I have ubuntu.tar in C:\users\ubuntu.tar and I want to name it new_ubuntu and have it stored in C:\users\new_ubuntu
`wsl -import new_ubuntu C:\users\new_ubuntu C:\users\ubuntu.tar`

**Reply by Nivethanb** [reply]
I want 22.04 image(.tar) file locally. When ever I try directly from machine it's re redirecting to MS store.

**Reply by lane-ftw** [reply]
Oh, if you can download it from the store you can follow [this guide.](https://4sysops.com/archives/export-and-import-windows-subsystem-for-linux-wsl/)
Otherwise here's the [direct downloads. ](https://cloud-images.ubuntu.com/wsl/)

**Reply by Nivethanb** [reply]
Thanks for the update ,,
we downloaded tart.gz file from git and successfully imported.
after that not getting start

![image](https://github.com/microsoft/WSL/assets/139915329/02bee2ec-0f70-472a-8e26-f33f2d939c41)

**Comment by elsaco** [comment]
@Nivethanb start with a Ubuntu cloud image. Current is https://cloud-images.ubuntu.com/wsl/jammy/current/ubuntu-jammy-wsl-amd64-wsl.rootfs.tar.gz

After download use `wsl --import` to install it. These are Ubuntu images build specifically for WSL.

**Reply by Nivethanb** [reply]
@elsaco now working fine, im not able start WSL genie -s followed below steps , 
"/bin/bash: line 1: genie: command not found"
https://gist.github.com/djfdyuruiry/6720faa3f9fc59bfdf6284ee1f41f950
